.gitignore

```diff
@@ -4,3 +4,4 @@ __dummy.html
 *.o
 *.obj
 *.a
+chipmunkd.lib
```

Comment:
lets just make this `*.lib`, no `.lib` files should be checked in. Also, you managed to commit your `chipmunkd.lib` anyways, could you remove that?